server/controllers/profile.controller.js

```diff
@@ -71,6 +71,68 @@ class ProfileController {
     delete dataValues.password;
     return Response(res, 200, 'Profile updated successfully.', [dataValues]);
   }
+
+  /**
+     * @description follows another user
+     * @param {object} req
+     * @param {object} res
+     * @returns {object} user that has been followed
+     */
+  static async followUser(req, res) {
+    const { username } = req.params;
+    const { id } = req.user;
+    const requestingFollowership = await User.findByPk(id);
+    const personToBeFollowed = await User.findOne({ where: { username } });
```

Comment:
`personToBeFollowed` - Poor variable naming. Use `follower` and `followee`. That makes more sense

Reply:
Ok will implement